internal/wasm/sdk/opa/pool.go

```diff
@@ -46,7 +48,10 @@ func newPool(poolSize, memoryMinPages, memoryMaxPages uint32) *pool {
 // Acquire obtains a VM from the pool, waiting if all VMms are in use
 // and building one as necessary. Returns either ErrNotReady or
 // ErrInternal if an error.
-func (p *pool) Acquire(ctx context.Context) (*vm, error) {
+func (p *pool) Acquire(ctx context.Context, metrics metrics.Metrics) (*vm, error) {
+	metrics.Timer("opa_wasm_pool_acquire_vm").Start()
+	defer metrics.Timer("opa_wasm_pool_acquire_vm").Stop()
```

Comment:
nit; Let's drop the `opa_` prefix on the metric keys. We don't do that anywhere else.

Reply:
Will do. The prefix was just from the package name 😄  but I do like a "less is more" approach to these.